lib/aws/active_job/sqs/configuration.rb

```diff
@@ -17,20 +19,46 @@ class Configuration
           excluded_deduplication_keys: ['job_id']
         }.freeze
 
-        # @api private
-        attr_accessor :queues, :max_messages, :visibility_timeout,
-                      :shutdown_timeout, :client, :logger,
-                      :async_queue_error_handler, :message_group_id
+        GLOBAL_ENV_CONFIGS = %i[
+          config_file threads backpressure
+          max_messages shutdown_timeout
+          visibility_timeout message_group_id
+        ].freeze
 
-        attr_reader :excluded_deduplication_keys
+        QUEUE_ENV_CONFIGS = %i[
+          url max_messages
+          visibility_timeout message_group_id
+        ].freeze
 
         # Don't use this method directly: Configuration is a singleton class, use
         # +Aws::ActiveJob::SQS.config+ to access the singleton config.
         #
+        # This class provides a Configuration object for AWS ActiveJob
```

Comment:
Is this at the wrong scope? A lot of the documentation belongs onto the class and not on the initialize method.